shell/components/form/ResourceQuota/ProjectRow.vue

```diff
@@ -46,7 +46,7 @@ export default {
 </script>
 <template>
   <div v-if="typeOption" class="row">
-    <Select class="mr-10" :mode="mode" :value="type" :options="types" @input="updateType($event)" />
+    <Select class="mr-10 resource-type" :mode="mode" :value="type" :options="types" @input="updateType($event)" />

```

Comment:
I don't know why UnitInput renders at 61px when it has no label - it should render smaller if there is no label. I added a CompactMode mixin where you can force it to be small or large - had a quick try but this is not working for UnitInput.

Feels like UnitInput should render smaller when there is no label and then you would not need the workaround for the resource-type class.

Reply:
After some fun I tracked this down to a bad merge of `LabeledInput` in https://github.com/rancher/dashboard/commit/50aed3eb9ebc1b1204979956dd74d408451e4ff3#diff-7e2477017dd4884bb65dab37ba1cd13997b43e6abc8aa57c1f5f6feba00a2cb3L166 that removed some classes and styles brought in with the original compact mode mixin change https://github.com/rancher/dashboard/commit/d781a568fc4f41523fc68461a2efb24366df3ccd#diff-442567f56ce9ffeaf98352b955051ba805ec143472912325881f0994b22cbd5eR145

I've now pushed a fix and updated the PR description with screens.

This does affect anywhere LabeledInput sans label is used, however this is what the code was in March... so only newer features might need updating

Reply:
There's a number of places `$input-height` (61px) is used. I'll try to mop those up in another PR